db/migrations/20180927105539_add_broker_name_and_guid_to_service_usage_event.rb

```diff
@@ -0,0 +1,12 @@
+Sequel.migration do
+  change do
+    # As service broker name and guid are stored as 'String' (with no length
+    # limit specified) in the service_brokers table, we have to use the same
+    # field type here.
+
+    # rubocop:disable Migration/IncludeStringSize
+    add_column :service_usage_events, :service_broker_name, String, null: true
+    add_column :service_usage_events, :service_broker_guid, String, null: true
```

Comment:
@tcdowney Thank you for the comment.

Do you suggest we should add size restriction  of 255 chars only to the `service_broker_guid` and leave `service_broker_name ` as `String`/`text`?

This will make `service_broker_guid` field inconsistent with its definition in `service_brokers` table. 
Currently they are both of type `String` with `text => true`.
```
create_table(:service_brokers, :ignore_index_errors=>true) do
      String :guid, :text=>true, :null=>false
      String :name, :text=>true, :null=>false
end
```

Reply:
Yeah, I'd rather not add new `String`/`text` columns if we can help it.

I agree we have `service_broker_name` as  since we have no idea what names folks might have placed in there on Postgres -- but `service_broker_guid` we know that we control and set [here](`String`/`text`) so I'd prefer if that one had a limit.

Reply:
> We'll add size 255 to service_broker_name field.

Err... `service_broker_guid`, right? 😛

Reply:
Oops, my bad, yes I meant `service_broker_guid`.